src/core/test/system/base_system.sol

```diff
@@ -57,24 +67,32 @@ contract BaseSystemTest is TestSetup, Math, DSTest {
             juniorInvestor = new Investor(address(juniorOperator), currency_, address(juniorERC20));
             juniorInvestor_ = address(juniorInvestor);
             WhitelistOperator juniorOperator = WhitelistOperator(address(juniorOperator));
-            juniorOperator.relyInvestor(juniorInvestor_);  
+            juniorOperator.relyInvestor(juniorInvestor_);
     }
 
     function lockNFT(uint loanId, address usr) public {
         Borrower(usr).approveNFT(collateralNFT, address(shelf));
         Borrower(usr).lock(loanId);
-    } 
+    }
 
     function transferNFT(address sender, address recipient, uint tokenId) public {
         Borrower(sender).approveNFT(collateralNFT, address(this));
         collateralNFT.transferFrom(sender, recipient, tokenId);
     }
 
     // helpers borrower
-    function issueNFT(address usr) public returns (uint, bytes32) {
-        uint tokenId = collateralNFT.issue(usr);
-        bytes32 lookupId = keccak256(abi.encodePacked(collateralNFT_, tokenId));
         // user approves shelf too lock NFT
+    function createSeniorInvestor() public {
+        seniorInvestor = new Investor(address(seniorOperator), currency_, address(seniorERC20));
```

Comment:
I will take out this code in my PR, I updated the createTestUsers method to handle this, but I think it's fine to leave in now